lib/nylas/event.rb

```diff
@@ -27,6 +27,7 @@ class Event
     attribute :status, :string
     attribute :title, :string
     attribute :when, :when
+    attribute :metadata, :hash
```

Comment:
Should we make a new type called `Metadata` or it can contain anything `key`, `value` pairs?

Reply:
`Metadata` is just a bunch of string key-value pairs. It can be anything the user wants it to be, it's abstract in a sense. I think we should leave it as is instead of creating a type for it.